raiwidgets/raiwidgets/_input_processing.py

```diff
@@ -103,6 +103,36 @@ def _convert_to_string_list_dict(
     return result
 
 
+def _safe_serializer(o):
+    """
```

Comment:
why can't the method below be used?  It looks like there is duplicate code with _serialize_json_safe

Reply:
line 133 is the key, we need to return __dict__ in order for json.dump to serialize and class object.

Reply:
refactored.